src/flavors/__tests__/fixtures.ts

```diff
@@ -69,7 +80,7 @@ export function createFixtures(
         requiredIntegerValue!: number;
 
         @IsNested({
-            nested: NestedRequiredExample,
+            type: NestedRequiredExample,
```

Comment:
For consistency, I'm now using `type` instead of `nested`. This is a break change, but since there's very little usage so far, I think it's ok.